### ISSUE TITLE
Date Choice

### ISSUE THREAD

**Issue by Spankythemusical** [custom-components/sensor.sonarr_upcoming_media#8]
No problem, and a possibility that this option might already be possible.  Is there a way to not show the current date?  I don’t think that it’s possible via Sonarr, but I could be wrong.  As I use both Sonarr Upcoming as well as Plex Recently Added, I’m usually shown the same item twice, as the item will be added to Plex while still showing as “upcoming” in Sonarr.  While this is by no means a big deal, it’s just something that would make things look nicer.

**Comment by maykar** [comment]
You can customize all text to your liking:

https://github.com/custom-cards/upcoming-media-card#text-content

**Comment by Spankythemusical** [comment]
Maybe I didn’t explain it correctly, or maybe I’m just not understayhow to implement what I’m attempting.  Hopefully my photo uploads, as I’m using my phone at the moment.  If it does, the left side is Sonarr Upcoming and the right side is Plex Recently Added.  Both show the same item, date and everything.  If I could make the Sonarr Upcoming show items starting from tomorrow, and skipping the current date, this would be alleviated. 
![4AAB68DD-7B4C-40EF-87CF-3B9C1F5B036A](https://user-images.githubusercontent.com/38551704/58135608-0456ae80-7c23-11e9-947b-1697f3e7a1fb.png)

**Comment by maykar** [comment]
Ah, yes, sorry for the misunderstanding. I thought when you said "Is there a way to not show the current date?" you were talking about the displayed text. Thanks for the clarification.

Let me know if this works for you, but I think a better option might be to add the option to hide downloaded episodes for the sonarr and radarr component.

**Comment by Spankythemusical** [comment]
That would be great.  I've been looking through the Sonarr documents to see if there's a way to remove objects from the upcoming list once they've been downloaded, but I have yet to be successful.  The hide option might be best in case an object is scheduled but has not been downloaded.   Like I said before, it's not a huge deal, just something that will clean up the view a bit.

**Comment by maykar** [comment]
Added the options hide_flagged and hide_unflagged to [upcoming-media-card](https://github.com/custom-cards/upcoming-media-card). Thank you!

**Comment by Spankythemusical** [comment]
I just want to give a massive thank you for this!!!  It is exactly what I was hoping for!!

**Comment by Spankythemusical** [comment]
I hate to keep bothering you with this, but I've realized an issue that I can't figure out.  When I added   `hide_flagged: true`, it did exactly what I was hoping.  In the app.  Unfortunately, it does not change anything, either through the Home Assistant Companion Beta app, nor via a web browser.  I have tried both Chrome and Firefox.  I know that the flags are seen, since using `flag_color` does change the color.  Any ideas?
